packages/database-editor/src/components/ImportWizard/Timeline.tsx

```diff
@@ -0,0 +1,53 @@
+import { Flex, Label } from '@axonivy/ui-components';
+import { Fragment } from 'react';
+import type { ImportPage } from './ImportWizard';
+import './Timeline.css';
+
+export const Timeline = ({ pages, active, setActive }: { pages: Array<ImportPage>; active: number; setActive: (i: number) => void }) => {
+  return (
+    <Flex className='timeline' direction='row' justifyContent='center'>
+      {pages.map((page, i) => (
+        <Fragment key={i}>
+          <TimelineItem title={page.title} active={i === active} passed={i < active} index={i} setActive={setActive} />
+          {i < pages.length - 1 && <Connector active={i < active} />}
+        </Fragment>
+      ))}
+    </Flex>
+  );
+};
+
+const TimelineItem = ({
+  title,
+  active,
+  passed,
+  index,
+  setActive
+}: {
+  title: string;
+  active: boolean;
+  passed: boolean;
+  index: number;
+  setActive: (i: number) => void;
+}) => {
+  return (
+    <Flex
+      className={`timeline-item${activeOrPassed(active, passed)}`}
```

Comment:
you can use the cn util from ui-components
```suggestion
      className={cn('timeline-item', active && 'active', passed && 'passed')}
```